experimental/cloudstrapper/README.md

```diff
@@ -272,7 +272,10 @@
         - Add ```--skip-tag clearSSHKeys``` if you want to keep ssh keys on the instance
     ```
 
-    ansible-playbook ami-configure.yaml -i <DynamicInventoryFile> -e "dirLocalInventory=<WORK_DIR>" -e "aminode=tag_Name_<buildGwTagName>" -e "ansible_python_interpreter=/usr/bin/python3" -u ubuntu --skip-tags clearSSHKeys
+    ansible-playbook ami-configure.yaml -i "<WORK_DIR>/common_instance_aws_ec2.yaml" -e "dirLocalInventory=<WORK_DIR>" -e "aminode=tag_Name_<buildGwTagName>" -e "ansible_python_interpreter=/usr/bin/python3" -u ubuntu --skip-tags clearSSHKeys
+
+    Example:
```

Comment:
Right, thanks!

Reply:
> * If this is good and valid documentation, we should think about moving it from `experimental` to `docs` or some other meaningful place.

I would keep the cloudstrapper in the experimental folder for now, and the README close to the code, so I'd keep it where it is for now. I think the other document that you mentioned is targeted to people actually deploying Magma with the Cloudstrapper while this README is targeted to developers working on it.

>     * git (or Github) does not understand the renaming of the file, which makes the diff rather hard to review. Maybe one could try a `git mv` again?

Maybe in this case it actually makes it easier ;) Looking at the diff isn't that helpful, because there is more diff than content.